virtualfish/virtual.fish

```diff
@@ -440,9 +442,20 @@ function __vf_help --description "Print VirtualFish usage information"
     echo
     echo "Available commands:"
     echo
-    for sc in (functions -a | sed -n '/__vf_/{s///g;p;}')
+
```

Comment:
Perhaps put a comment just above this stanza that explains its purpose? Something like:

    # Dynamically calculate column spacing, based on longest subcommand